docs/TOKENS.md

```diff
@@ -13,133 +13,141 @@ Spoilers. My love for Astro has grown deeper.
 
 ```ts
 import type { APIContext } from "astro";
 
-export async function GET({ params }: APIContext) {
-  return new Response(
-    JSON.stringify({
-      rtcToken: "token",
-    })
-  )
-}
-```
-
-## CORS
-We first need to set up our headers so that we don't encounter any CORS issues when using our backend.
-
-```ts
-export async function GET({ params }: APIContext) {
-    const headers = new Headers({
-        "Access-Control-Allow-Origin": "*",
-        "Access-Control-Allow-Methods": "GET, OPTIONS",
-    });
+export async function POST({ request }: APIContext) {
+    const { channel, role, uid, expireTime } = await request.json()
 
     return new Response(JSON.stringify({
-        rtcToken: "token"
-    }), { headers })
+        token: "token"
+    }), { status: 200 })
 }
 ```
 
+## Response Helper Functions
+We will abstract away the Response objects in a `utils/sendResponse.ts` file to simplify our endpoint. 
+
+First, we need to set up our headers to avoid any CORS issues when using our backend.
+
 * "Access-Control-Allow-Origin": "*" allows all domains to access the server, which is helpful for APIs or during development.
-* "Access-Control-Allow-Methods": "GET, OPTIONS" specifies that only GET and OPTIONS methods are permitted, controlling how other sites can interact with the server.
+* "Access-Control-Allow-Methods": "POST, OPTIONS" specifies that only POST and OPTIONS methods are permitted, controlling how other sites can interact with the server.
 
 These headers will allow our API to work well in production and development by allowing all domains. For production, you should limit only the origins that should be allowed.
 
+Then we craft a `sendBadRequest` function with a status `400` and a `sendSuccessfulResponse` function with a status `200` that uses the headers we defined.
+
+```ts
+const headers = new Headers({
+    "Access-Control-Allow-Origin": "*",
+    "Access-Control-Allow-Methods": "POST, OPTIONS",
+});
+
+export const sendBadRequest = (reason: string) => {
+    return new Response(reason, { status: 400, headers });
+}
+
+export const sendSuccessfulResponse = (data: any) => {
+    return new Response(JSON.stringify(data), { status: 200, headers });
+}
+```
+
 ## Input Checks
-The token generation function requires the `channel`, `role`, and `uid`. These are used to ensure that the token includes correct authentication and privileges. To ensure we pass accurate data, check that `channel` and `uid` are not empty. If they are, return a Bad Request Error.
+The token generation function requires `channel`, `role`,  `uid`, and `expireTime`. These are used to ensure that the token includes correct authentication and privileges. To ensure we pass accurate data, check that `channel`, `uid`, and `expireTime` are not empty. If they are, return a Bad Request using our helper function.
 
 The role determines what type of access the user requesting a token should have. There are two possible roles:
```

Comment:
Add ` to role, becasue it isn't clear that you are still referencing the `role` from the previous paragraph.